npm/test/clientfilemetataupload.test.ts

```diff
@@ -195,6 +195,30 @@ test("createMetadataInputBatches generates batches of the defined size", () => {
   expect(result[1]).toEqual([input4, input5])
 })
 
+test("createMetadataInputsAndFileMap returns metadata inputs and file map", () => {
+  const metadata: IFileMetadata[] = [mockMetadata1, mockMetadata2]
+  const client = new GraphqlClient("https://test.im", mockKeycloakInstance)
+  const uploadMetadata = new ClientFileMetadataUpload(client)
+
+  const { metadataInputs, matchFileMap } =
+    uploadMetadata.createMetadataInputsAndFileMap(metadata)
+
+  expect(metadataInputs).toHaveLength(metadata.length)
```

Comment:
Minor thing, but can you make this `.toHaveLength(2)`? It makes it more obvious that we have a strong expectation about what the value should be, and it makes it easier to avoid bugs where we accidentally use empty arrays so none of the assertions below get run.